### PR TITLE
wip: add cert-manager demo

### PR DESCRIPTION
**Pull request by njhale**
## Summary

Add cert-manager demo script.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -127,6 +127,10 @@ demo-kubecon: build ## Run the KubeCon demo.
 demo-ingress: build ## Run the Ingress demo.
 	cd contrib/demo && ./runDemoScripts.sh ingress
 
+.PHONY: demo-cert-manager
+demo-ingress: build ## Run the cert-manager demo.
```
Comment (sttts) [inline Makefile:131]: wrong target

**Comment by s-urbaniak** [comment]
@njhale out of curiosity, how are you deploying cert-manager. We're the team responsible for integrating it to openshift 🤓 cc @slaskawi

**Comment by njhale** [comment]
> @njhale out of curiosity, how are you deploying cert-manager. We're the team responsible for integrating it to openshift 🤓 cc @slaskawi

@s-urbaniak check out #763 for a working demo deploying cert-manager. TL;DR, I believe the manifests used in that branch are the result of evaluating the Helm chart provided in cert-manager's docs + an image containing a [kcp aware cert-manager fork](https://github.com/fabianvf/cert-manager/pull/1).

**Comment by njhale** [comment]
Superseded by #763

**Pull request closed without merging**